add(Install, HW and SW Reqts): add instructions to patch Chrome 60

ugaston opened the pull request:
Versions: Proposal for 7.5 (should be adapted for all versions starting from 6.2)

Set of instructions to patch Bonita BPM Platform for Chrome 60+ versions, both for Subscription and Community users.

_Reference bug: **BS-16954**- Portal incompatible with Chrome 60 (released 25 Jul 2017)

ugaston committed:
add(Install, HW and SW Reqts): add instructions to patch Chrome 60

Versions: Proposal for 7.5 (should be adapted for all versions starting from 6.2)

Set of instructions to patch Bonita Platform for Chrome 60+ versions, both for Subscription and Community users.

Reference bug: *BS-16954* - Portal incompatible with Chrome 60 (released 25 Jul 2017)
Escalation reference: *2017-07-31 - University of Queensland - Risk Action Plan* (https://bonitasoft.atlassian.net/wiki/spaces/0015700001oVd76/pages/127393923/2017-07-31+-+University+of+Queensland+-+Risk+action+plan+-+Chrome+60#space-menu-link-content)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bonitasoft/bonita-doc?pullRequest=370) <br/>All committers have signed the CLA.

tbouffard commented:
@ugaston can you please sign the CLA?

tbouffard commented:
@ugaston does this modification should also be applied to doc 7.3 and 7.4? I have just seen you created the pull request on the 7.5 branch.

ugaston commented:
Basically, the procedure to patch the portal has to be documented for all versions starting from 6.2.

This is (version-specific details notwithstanding) the procedure to communicate, but the Hardware and Software Requirements page has evolved slightly over the versions, so the actual placement of the procedure may need adapting for the previous versions.

abirembaut commented on md/hardware-and-software-requirements.md:
````diff
@@ -50,5 +50,74 @@ Notes:
+
+1. In an installed Bonita bundle, edit file 'server/webapps/bonita/portal/scripts/includes/common.js':
+   1. Add the code below after or before the existing code:
+   ```java
````
should be javascript

abirembaut reviewed (changes requested):
LGTM
Just one small comment

ugaston committed:
Update hardware-and-software-requirements.md

Replace "java" by "javascript" for the code snippet's formatting, following Anthony's comment.

ugaston commented:
Thank you for your comment, Anthony. Good catch.
I have now replaced "java" with "javascript", as suggested.

laurentleseigneur commented on md/hardware-and-software-requirements.md:
````diff
@@ -50,5 +50,74 @@ Notes:
+1. Unzip the exported file 'portal-theme.zip'.
+1. Edit 'BonitaConsole.html' file:
+   1. In the `<head>` section, add the code below:
+   ```xml
````
this script block is `javascript`, not `xml`

sperfect commented on md/hardware-and-software-requirements.md:
```diff
@@ -50,5 +50,74 @@ Notes:
+      // Monkey Patch xhr
+      // Due to a specification change in the xhr.getAllResponseHeaders method Bonita Portal does not behave as expected 
+      // in browsers that implement this new specification (currently only Chrome >60).
+      // This patch fixes xhr.getAllResponseHeaders unwanted behavior whithin Bonita Portal context
```
within

ugaston committed:
Update hardware-and-software-requirements.md

Taking the last two review comments into account:
- xml --> javascript
- whithin --> within

ugaston commented:
Thank you, Laurent and sperfect (?) for your comments.
My last commit should have taken good care of those.

abirembaut committed:
Move database charset conf to the top

abirembaut committed:
Move database charset conf to the top

abirembaut commented:
LGTM
